src/components/Li.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+  <li v-if="this.$slots.default" class="slot">
+    <slot />
+  </li>
+  <li v-else :class="{ disabled: !options.link.link && !options.link.onClick, options: true }">
+    <Link :options="options.link">
+      <div>
+        <p class="first-value">{{options.firstValue}}</p>
+        <p class="second-value" v-if="options.secondValue">{{options.secondValue}}</p>
+      </div>
+      <p class="third-value" v-if="options.thirdValue">{{options.thirdValue}}</p>
+    </Link>
+  </li>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Prop } from 'vue-property-decorator';
+import Link, { LinkOptions } from '@/components/Link.vue';
+
+export interface LiOptions {
+  id: string;
+  firstValue: string;
+  secondValue?: string;
+  thirdValue?: string;
+  link: LinkOptions;
+}
+
+@Component({
+  components: {
+    Link,
+  },
+})
+export default class Li extends Vue {
+  // region public static methods
+  // endregion
+
+  // region private static methods
+  // endregion
+
+  // region public members
+  @Prop(Object)
+  public options?: LiOptions;
```

Comment:
Why is it called `options`? To me, it seems more like a `ListObject`, `ListElement`. (And it implies plural, where it only includes one link, so it is rather one option.)

Reply:
Even though `Li` refers to the HTML element, I'd prefer non-abbreviated names. I need to look at the surrounding code to understand what `LiOptions` might stand for. Why not just call the class and interface `ListEntry` or `ListElement`?

Reply:
Renamed that to `ListEntry`